pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -255,6 +255,7 @@ def _postsolve(self):
 
                     results.solver.time = log_dict['solving_time']
                     results.solver.gap = log_dict['gap']
+                    results.solver.nodes = log_dict['solving_nodes']
```

Comment:
I think `results.solver.node_count` might be a clearer name, but I don't have super strong feelings about it.